### PR TITLE
Adding missing values to network role

### PR DESCRIPTION
**Pull request by fao89**
https://github.com/openstack-k8s-operators/edpm-ansible/blob/main/edpm_ansible/playbooks/deploy-edpm-network-configure.yml

### CODE REVIEW

**Review by fultonj — COMMENTED** [review]
+2

File: pkg/deployment/network.go [hunk 1]
```diff
@@ -30,16 +30,19 @@ import (
 func ConfigureNetwork(ctx context.Context, helper *helper.Helper, obj client.Object, sshKeySecret string, inventoryConfigMap string) error {
 
 	role := ansibleeev1alpha1.Role{
-		Name:     "edpm_network_config",
-		Hosts:    "all",
-		Strategy: "linear",
+		Name:        "edpm_network_config",
+		Hosts:       "all",
+		Strategy:    "linear",
```
Comment (fao89) [inline pkg/deployment/network.go:35]: should this be `free`? 
https://github.com/openstack-k8s-operators/edpm-ansible/blob/main/edpm_ansible/playbooks/deploy-edpm-network-configure.yml#L19

Comment (slagle) [reply pkg/deployment/network.go:35]: No, I'd prefer to keep the strategy as linear. It makes things much easier to debug when looking at the ansible log. Unless we need to optimize, I'd say stick with linear.

File: pkg/deployment/network.go [hunk 2]
```diff
@@ -30,16 +30,19 @@ import (
...
+		GatherFacts: false,
 		Tasks: []ansibleeev1alpha1.Task{
 			{
 				Name: "import edpm_network_config",
 				ImportRole: ansibleeev1alpha1.ImportRole{
 					Name:      "edpm_network_config",
 					TasksFrom: "main.yml",
 				},
+				Vars: "  edpm_network_config_with_ansible: true",
```
Comment (slagle) [inline pkg/deployment/network.go:44]: The var isn't really needed as it's already defaulted to true

**Review by slagle — CHANGES_REQUESTED** [review]

**Pull request closed without merging**